### PR TITLE
Pr/extend manpage

### PR DESCRIPTION
**Pull request by uli42**
some fixes and improvements for nxagent's manpage

This also contains the solution to issue #430

### CODE REVIEW

File: nx-X11/lib/X11/Imakefile [hunk 1]
```diff
@@ -891,6 +891,13 @@ OBJS3 = \
 	xim_trans.o \
 	$(XKBOBJS) $(LCOBJS)
 
+SHAREDLCDEPSYMSRCS = XlcDL.c XlcSL.c
+
+SHAREDI18NSTATICSRCS = \
+        XDefaultIMIF.c XDefaultOMIF.c
+
+STATICLCDEPSYMSRCS =
```
Comment (sunweaver) [inline nx-X11/lib/X11/Imakefile]: @uli42: Why do we want to re-introduce 'XDefaultIMIF.c' and 'XDefaultOMIF.c'??? Not getting the point here. Please comment and improve the commit message, if you are really intending to revert this.

File: nx-X11/programs/Xserver/hw/nxagent/man/nxagent.1 [hunk 2]
```diff
@@ -536,6 +540,14 @@ means that all of these checks are essentially
 deactivated. This is a very bad idea.
 .RE
 
+If you want to use nxagent as a replacement for xnest or Xephyr you
```
Comment (sunweaver) [inline nx-X11/programs/Xserver/hw/nxagent/man/nxagent.1]: s/xnest/Xnest/

File: nx-X11/programs/Xserver/hw/nxagent/man/nxagent.1 [hunk 3]
```diff
@@ -540,7 +541,7 @@ means that all of these checks are essentially
 deactivated. This is a very bad idea.
 .RE
 
-If you want to use nxagent as a replacement for xnest or Xephyr you
+If you want to use nxagent as a replacement for Xnest or Xephyr you
```
Comment (sunweaver) [inline nx-X11/programs/Xserver/hw/nxagent/man/nxagent.1]: please merge the typo fix into the prev commit...

**Review by sunweaver — CHANGES_REQUESTED** [review]
@uli42: thanks for improving the nxagent man page. I left some comments, please amend.

Please also explain and document your reasoning for the first commit regarding the XlibI18n reverse commit. 

Ideally, the man page commits would be in this PR and the reverse commit for XlibI18n in a second PR.

File: nx-X11/lib/X11/Imakefile (hunk 1 above)
Comment (uli42) [reply nx-X11/lib/X11/Imakefile]: ??? Looks like I have done something wrong. This revert belongs to my static stuff: https://github.com/uli42/nx-libs/tree/nxagent_static

I'll update this PR accordingly

**Comment by uli42** [comment]
ok, new push without the wrong commit

**Review by sunweaver — APPROVED** [review]

**Pull request merged**